### ISSUE TITLE
"CTS profile match: fail" still after installing PlayIntegrityFix

### ISSUE THREAD

**Issue by rudolphos** [chiteroman/PlayIntegrityFix#284]
- Android 11, LineageOS
- Magisk app 27.0
- Magisk 27.0

Modules in Magisk: GMS Doze, Advanced charging controller (ACC), NoStorageRestrict, **Play Integrity Fix**, Shamiko
Magisk Zygist: enabled
Magisk enforce deny list: disabled

Disabled Shamiko, rebooted, the same issue. **CTS profile match: fail** when using YASNAC.

YASNAC json:
```
{
  "apkCertificateDigestSha256": [
    "CFMi+R9OCRrIqon0WdOwQgA4I9aG0m1sYPZuUf5a2ns="
  ],
  "apkDigestSha256": "6eSwd6Ed+f4Pab9sXTvk+hqbA8VFtfdCnYlznf+zIbc=",
  "apkPackageName": "rikka.safetynetchecker",
  "basicIntegrity": true,
  "ctsProfileMatch": false,
  "evaluationType": "BASIC",
  "nonce": "...",
  "timestampMs": 1709822154775,
  "deprecationInformation": "The app is allowlisted to use the SafetyNet Attestation API until the full turndown: https://g.co/play/safetynet-timeline."
}
```

**Comment by rudolphos** [comment]
I am currently using PIF 15.1 and turns out it was necessary to enable "enforce denylist" which was contraindicated in Shamiko.
I can now use one particular app as a result, but still can't pass CTS profile match, shows: FAIL.

**Comment by Dachoni03** [comment]
The same here

**Comment by Johndjk** [comment]
I'm not a technical person so I don't know if it's redundant how to put my information here but I had the same error:

{
  "apkCertificateDigestSha256": [
    "CFMi+R9OCRrIqon0WdOwQgA4I9aG0m1sYPZuUf5a2ns="
  ],
  "apkDigestSha256": "6eSwd6Ed+f4Pab9sXTvk+hqbA8VFtfdCnYlznf+zIbc=",
  "apkPackageName": "rikka.safetynetchecker",
  "basicIntegrity": true,
  "ctsProfileMatch": false,
  "evaluationType": "BASIC",
  "nonce": "....",
"timestampMs": 1709964209795,
  "deprecationInformation": "The app is allowlisted to use the SafetyNet Attestation API until the full turndown: https://g.co/play/safetynet-timeline."

What'll it be?

**Comment by Dachoni03** [comment]
Yeah we need new way to bypass SafetyNet, but for now there is no such way unfortunately.

**Comment by apprehensions** [comment]
Same here

**Comment by chiteroman** [comment]
LineageOS string in kernel name is banned, check.

**Comment by apprehensions** [comment]
`5.10.177-android13-4-00003-ga7208022a7ea-ab1081582`

No...?
